src/sentry/api/event_search.py

```diff
@@ -820,6 +832,17 @@ def to_list(value):
         # Deprecated alias, use `group_ids` instead
         if "issue.id" in params:
             kwargs["group_ids"] = to_list(params["issue.id"])
+        if "issue" in params:
```

Comment:
When is this used if you also have `issue` being handled as a condition?

Reply:
I'm not really sure why this is here. I added it here as well because issue.id is here. I was just trying to make sure I was handling every case that issue.id might get used.

Reply:
We should be accepting `issue` in the params dictionary though as that should be reserved for global selection header parameters. I'm not entirely sure why `issue.id` is here either.

Reply:
We shouldn't add `issue` here and should also track down where `issue.id` is being used and convert that into conditions as well. Having to manage filter_keys + conditions separately should be something we push down the the lower-level snuba functions or query abstraction.

Reply:
I'll remove `issue` but it's probably a separate PR to dig through all the different things that use this file to figure out which one is sending `issue.id`.